Documentation/BuildInstructionsLadybird.md

```
@@ -67,36 +67,22 @@ sudo add-apt-repository ppa:ubuntu-toolchain-r/test
-```
-
-Optionally, install the PulseAudio headers for audio playback support:
-
-```
-sudo pacman -S libpulse
+sudo pacman -S --needed autoconf-archive base-devel ccache cmake curl git less libgl libpulse nasm ninja python qt6-base qt6-positioning qt6-tools ttf-liberation tar unzip zip
 ```
 
 ### Fedora or derivatives:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)